azure-auth-helper/pom.xml

```diff
@@ -8,7 +8,7 @@
         <relativePath>../azure-maven-plugins-pom</relativePath>
     </parent>
     <artifactId>azure-auth-helper</artifactId>
-    <version>0.2.0-SNAPSHOT</version>
+    <version>0.3.0-SNAPSHOT</version>
```

Comment:
we have not released 0.2.0, keep 0.2.0-SNAPSHOT?

Reply:
Fixed